src/accordion/accordion.tsx

```diff
@@ -13,10 +13,11 @@ const AccordionBase = ({
     children,
     title,
     enableExpandAll = true,
+    initialExpandAll = true,
     showTitleInMobile = false,
     className,
 }: AccordionProps): JSX.Element => {
-    const [expandAll, setExpandAll] = useState<boolean>(true);
+    const [expandAll, setExpandAll] = useState<boolean>(initialExpandAll);
```

Comment:
We've decided to allow the expandAll props to be passed down. Hence for this to rename state name to `expandAllState` and also add a `useEffect` for any `expandAll` prop changes. 

Do remember to document this new prop also.

Reply:
Sorry after thinking it carefully, we'll forget about the `useEffect`. Let's just use the new suggested prop name and leave everything as it is. Still remember to document the prop